### PR TITLE
Update some package homepage URLS

### PR DESCRIPTION
**Pull request by nboelte**
Hi,

we used the "homepage" URLS from spack packages in our documentation and our static page generator complained about some of them (like 404 errors) and we fixed them. I thought it might be helpful to upstream those changes.

Best Wishes,
Niklas

### CODE REVIEW

**Review by dacolombo — APPROVED** [review]
Modifications to relion's package.py looks good to me

File: var/spack/repos/builtin/packages/openbabel/package.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Openbabel(CMakePackage):
     search, convert, analyze, or store data from molecular modeling, chemistry,
     solid-state materials, biochemistry, or related areas."""
 
-    homepage = "https://openbabel.org/wiki/Main_Page"
+    homepage = "https://openbabel.org/docs/index.html"
```
Comment (RMeli) [inline var/spack/repos/builtin/packages/openbabel/package.py]:
```suggestion
    homepage = "https://openbabel.org/index.html"
```
Nice to see the new documentation live. I think the link above is better as `homepage`.

**Review by RMeli — CHANGES_REQUESTED** [review]

**Review by RMeli — APPROVED** [review]
OpenBabel LGTM, thanks.

**Review by snehring — APPROVED** [review]
pbmpi change is fine

**Review by tldahlgren — APPROVED** [review]
Confirmed all eight homepages exist.

**Review by AlexanderRichert-NOAA — APPROVED** [review]

**Comment by tldahlgren** [comment]
Clicking `Update branch` to pick up a fix that should resolve the CI problem.

**Review by bernhardkaindl — APPROVED** [review]

**Comment by bernhardkaindl** [comment]
Some jobs failed, but is apparently still running:
@spackbot re-run pipeline

**Comment by bernhardkaindl** [comment]
@spackbot re-run pipeline

**Comment by alecbcs** [comment]
@spackbot run pipeline

**Comment by bernhardkaindl** [comment]
@spackbot re-run pipeline

**Pull request merged**